src/sentry/replays/blueprints/api.md

```diff
@@ -608,3 +608,103 @@ Retrieve a collection of click events associated with a replay.
     ]
   }
   ```
+
+## Replays Viewed Status [/organizations/<organization_slug>/replays-viewed/]
```

Comment:
Nit:
```suggestion
## Replays Viewed Status [/organizations/<organization_slug>/replays/viewed-status/]
```

Reply:
I don't think we should do `/replays/viewed-status/` because it'll conflict with the existing `/replays/(?P<replay_id>[\w-]+)/` route that we have. If both routes are defined then the order they are defined will really matter, and it's not worth the hassle that we will have to deal with. It's a slightly less good url, but `/replays-viewed/` will help to avoid problems later.